lib/index.js

```diff
@@ -1 +1,17 @@
-module.exports = require('../native');
+const { ctrlc } = require('../native');
+const { join } = require('path');
+module.exports = {
+  ctrlc(pid) {
+    try {
```

Comment:
Is it worth checking `process.platform !== 'win32'` here and throw to avoid needing to compile a non-windows rust binary?